files/image_config/caclmgrd/caclmgrd

```diff
@@ -16,7 +16,10 @@ try:
     import subprocess
     import sys
     import syslog
-    from swsssdk import ConfigDBConnector
+
+    from sonic_py_common import device_info
+    from swsscommon import swsscommon
+    from swsssdk import SonicDBConfig, ConfigDBConnector
```

Comment:
swsscommon is meant to replace swsssdk completely. I'd rather not import both; I'd prefer to switch to swsscommon entirely, but maybe that's best suited for a separate PR.

Reply:
@jleveque . Create Issue#https://github.com/Azure/sonic-buildimage/issues/5219 for tracking this item.